loader/include/Geode/loader/Log.hpp

```diff
@@ -153,7 +153,7 @@ namespace geode {
             Log& operator=(Log&& l) = default;
             bool operator==(Log const& l);
 
-            std::string toString(bool logTime = true) const;
+            std::string toString(bool logTime = true, uint32_t nestLevel = 0) const;
```

Comment:
best you can do is 
```cpp
std::string toString(bool logTime = true) const;
std::string toString(bool logTime, uint32_t nestLevel) const;
```
 because otherwise it would be ambiguous